libs/platform/core/src/orchestration/app/app.model.ts

```diff
@@ -11,18 +8,11 @@ declare global {
   }
 }
 
-export interface ModularAppBuilderOptions {
-  injector?: Omit<InjectorOptions, 'providers'>;
-  components?: Partial<ComponentsOptions>;
-}
-
+// TODO: move out to the experience package
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface AppFeature {
```

Comment:
Why should `AppFeature` be moved to experience? It's a core part of the framework.